inductiva/api/methods.py

```diff
@@ -327,25 +363,63 @@ def invoke_api(method_name: str,
             else:
                 logging.info("Task failed.")
 
-        logging.info("Downloading output...")
-        output_zip_path = download_output(
-            api_instance=api_instance,
-            task_id=task_id,
-        )
-        logging.info("Output downloaded.")
+        result_list = download_output(api_instance, task_id, output_dir)
 
-    if output_dir is None:
-        output_dir = os.path.join(inductiva.output_dir, task_id)
+        if status == "failed":
+            raise RemoteExecutionError(f"""Remote execution failed.
+        Find more details in: \"{os.path.abspath(output_dir)}\".""")
 
-    logging.info("Extracting output ZIP file to \"%s\"...", output_dir)
-    result_list = extract_output(output_zip_path, output_dir)
-    logging.info("Output extracted.")
+    return unpack_output(result_list, output_dir, return_type)
 
-    if status == "failed":
-        raise RemoteExecutionError(f"""Remote execution failed.
-    Find more details in: \"{os.path.abspath(output_dir)}\".""")
 
-    return unpack_output(result_list, output_dir, return_type)
+def invoke_async_api(method_name: str, params, type_annotations: Dict[Any,
+                                                                      Type]):
+    """Perform a task asyc and remotely via Inductiva's Web API.
+
+    Submits a simulation async to the API and returns the task id.
+    The flow is similar to invoke_api, but it does not block until the task
+    is finished. 
+    The flow is summarized as follows:
+        1. Transform request params into the params used to
+        validate permission to execute the request.
+        2. Submit task via the "POST task/submit" endpoint.
+            Note: HTTP status code 400 informs the requested method is invalid,
+                and HTTP status code 403 informs that the user is not authorized
+                to post such request.
+        3. If the status returned by the previous HTTP request is
+            "pending-input", ZIP inputs and send them via
+            "POST task/{task_id}/input".
+        4. Return task_id and leaves the simulation running on the server.
```

Comment:
The simulation is not necessarily left running on the server. It can be waiting in the queue until resources become available.